README.md

```diff
@@ -160,6 +172,12 @@ This can be used to profile a Java process from the start.
 java -javaagent:ap-loader.jar=start,event=cpu,file=profile.html <java arguments>
 ```
 
+With JBang you can do:
+
+```sh
+java -javaagent:ap-loader@jvm-profiling-tools/ap-loader=start,event=cpu,file=profile.html <java arguments>
```

Comment:
I tried it and it failed, but maybe I haven't setup JBang correctly.

The README would suggest that the user would be getting more information on running ap-loader with JBang at the async-profiler GitHub page.

Maybe leave out this paragraph?

Reply:
doh - i typed it wrong ;)